tests/cylc-show/prereqs-outputs/suite.rc

```diff
@@ -49,11 +60,19 @@
         inherit = BAR, SHOW
         script = true
     [[baz]]
-       script = sleep 10
+        script = sleep 10
     [[baz2]]
         inherit = None, SHOW
+        script = """
+            cylc suite-state "${CYLC_SUITE_NAME}" \
+                -t baz -p 1 -S start --interval=1
+        """
     [[<m>]]
         inherit = None, RESET, SHOW
     [[trigger]]
         inherit = None, SHOW
-        script = cylc trigger ${CYLC_SUITE_NAME} baz.1; sleep 5
+        script = """
+            cylc trigger ${CYLC_SUITE_NAME} baz.1
+            cylc suite-state "${CYLC_SUITE_NAME}" \
+                -t baz -p 1 -S start --interval=1
+        """
```

Comment:
I am still none the wiser with this test!

Reply:
I'm not much better but it is at least stable.